### DISCUSSION TITLE
If you can build Zed for Windows, why aren't there binaries for it?

### DISCUSSION THREAD

**Discussion by denivic** [zed-industries/zed#9389] [Feature Requests]
Hello!

Zed looks pretty interesting and I'd love to try it. I have no experience with Rust, so my understanding may be flawed here, but if you are able to build it on Windows from the source code then how come there are no available binaries on the official site?

**Comment by SomeoneToIgnore** [comment]
Hello.

The official website provides official binaries, which Zed team considers ready for publishing.
The Windows build is not there yet: it does not support many features (no collaborative ones, for instance), may panic on certain actions or work unexpectedly otherwise.
For instance, Vim mode _started_ to work only yesterday, thanks to https://github.com/zed-industries/zed/pull/9314

I must say, I'm quite impressed with how fast the things are moving on Windows front, so it might be that we're having more links on the website before the end of the year.

**Reply by denivic** [reply]
Thank you for elaborating. I'm looking forward to testing the full version when it drops.

**Comment by felipetesc** [comment]
Hello. Changing a little bit from the subject. Is it possible to create a Windows Roadmap like the one from [Linux](https://github.com/zed-industries/zed/issues/7015) ? Since this is an open source project, to have some guidance from Zed's Team would be nice case we want to help to implement the windows's functionalities, and speed up the offering of windows binaries to be downloaded from the official website.

Thanks.

**Reply by Moshyfawn** [reply]
While Linux has a pretty straightforward path to official support, just like Windows at this point, it started as a community effort as opposed to the Zed team. I doubt it will be possible to provide any kind of roadmap at this point, as a lot of effort is currently focused on delivering Linux support.
